sceptre-example/config/lambda-stack-bootstrapless.yaml

```diff
@@ -18,7 +18,7 @@ template:
     file_asset_bucket_name: !stack_attr template_bucket_name
     # It can be useful to apply the same prefix as your template_key_prefix to ensure your
     # assets are namespaced similarly to the rest of Sceptre's uploaded artifacts.
-    file_asset_prefix: {{template_key_prefix}}/cdk-assets
+    file_asset_prefix: "{{template_key_prefix}}/cdk-assets"
```

Comment:
quoted to fix the following error..

```
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

sceptre-example/config/lambda-stack-bootstrapless.yaml
  21:47     error    syntax error: expected <block end>, but found '<scalar>' (syntax)
```

Reply:
Thanks. There are several that started failing on the main branch once I activated webhooks. 🤦 I didn't realize it wasn't running tests until after I merged. But we're not quite ready to deploy for the first time anyway, so I think we're still ok. We're still in "alpha". I'll give you 1000 Jon points if you manage to get all the tests passing.

Reply:
Are Jon points redeemable for anything?  Like one of those big stuffed animal from the circus?